_posts/en/newsletters/2022-03-30-newsletter.md

```diff
@@ -0,0 +1,176 @@
+  in [Newsletter #188][news188 gossip], LN protocol developers are
+  discussing how to revise the LN gossip protocol used to advertise
+  information about available payment channels.  In particular, this
+  week saw two active threads:
+
+    - *Major update:* in [response][osuntokun gossip2] to Rusty
+      Russell's [major update][russell gossip2] proposal from last month,
+      Olaoluwa Osuntokun repeatedly expressed concern with an aspect of
+      the proposal that would introduce plausible deniability in the
+      link between onchain funds and a specific LN channel.  That
+      capability would also make it easier for non-LN users to advertise
+      the existence of channels that might not actually exist, which
+      could degrade the ability for a spender to find a working path
+      across the network to the node receiving the funds.
+
+    - *Minor update:* Osuntokun [posted][osuntokun gossip1.1] a separate
+      proposal for a much smaller update to the gossip protocol aimed
+      mainly at allowing taproot-based channels.  The proposal uses
+      [MuSig2][topic musig] to allow a single signature to prove
+      authorization related to all four public keys involved (two node
+      identifier keys, two channel-spending keys) and would likely
+      require that the channel setup transaction be spendable using
+      MuSig2.
+
+        He also suggested that it might be useful to add an SPV partial
+        merkle branch proof to the channel announcement message.  This
+        would prove that the channel setup transaction was included in a
+        block, eliminating lightweight clients from having to download
+        the entire block containing the transaction in order to verify
+        its existence.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [What are the advantages or disadvantages to address reuse?]({{bse}}112955)
+  RedGrittyBrick and Pieter Wuille list the downsides of address reuse including
+  privacy and debatable concerns around pubkey exposure. Wuille goes on to note
+  that while generating new addresses has no incremental financial costs, it
+  does add complexity to wallet software, backups, and usability.
+
+- [What is a block-relay-only connection and what is it used for?]({{bse}}112828)
+  User vnprc describes block-relay-only connections as peers that relay block information
+  but not transactions or IP addresses of potential peers. These connections
```

Comment:
Strictly speaking, IP addresses are only a subset of the node addresses, since we also support other networks. Perhaps consider "node addresses" or disregard because it may be clearer in the context to say "IP addresses" than potentially having readers think that we're referring to Bitcoin addresses.